src/main/starlark/builtins_bzl/common/cc/cc_helper.bzl

```diff
@@ -346,7 +346,7 @@ OBJCPP_SOURCE = [".mm"]
 CLIF_INPUT_PROTO = [".ipb"]
 CLIF_OUTPUT_PROTO = [".opb"]
 CC_HEADER = [".h", ".hh", ".hpp", ".ipp", ".hxx", ".h++", ".inc", ".inl", ".tlh", ".tli", ".H", ".tcc"]
-ASSESMBLER_WITH_C_PREPROCESSOR = [".S"]
+ASSEMBLER_WITH_C_PREPROCESSOR = [".S"]
```

Comment:
There's a Google-internal reference to this symbol in Piper that needs to be updated after the next blaze release.
Could you please keep the old name as a duplicate here, along with the new name, and add this comment for me to track the cleanup?
TODO(b/345158656): Remove ASSESMBLER_WITH_C_PREPROCESSOR after next blaze release